frontend/apps/vara-man/src/feature/game/models/Game.ts

```diff
@@ -119,118 +127,135 @@ export class GameEngine {
     this.fogContext.scale(dpr, dpr);
 
     this.render();
-  }
+  };
+
+  private handleResize = () => {
+    if (this.resizeTimeout) {
+      clearTimeout(this.resizeTimeout);
+    }
+    this.resizeTimeout = window.setTimeout(() => {
+      this.resize();
+    }, 200);
+  };
 
   private initEventListeners() {
     window.addEventListener('keydown', this.handleKeyDown);
     window.addEventListener('keyup', this.handleKeyUp);
-    window.addEventListener('resize', this.resize.bind(this));
+    window.addEventListener('resize', this.handleResize);
   }
 
   public handleKeyDown = (event: { keyCode: number }) => {
     switch (event.keyCode) {
```

Comment:
nit: keyCode is [deprecated property](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).
Instead use [KeyboardEvent.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key)